### PR TITLE
Change location from London to Cambridge

### PR DESCRIPTION
**Pull request by stevelittlefish**

### CODE REVIEW

**Comment by salimfadhley** [comment]
It looks like you trimmed off the --- lines which need to come before and after the header.  Can you have a go at fixing that?

File: jobs/wegift_python_software_engineer.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (salimfadhley) [inline jobs/wegift_python_software_engineer.html]: This line is needed

File: jobs/wegift_senior_python_software_engineer.html [hunk 2]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (salimfadhley) [inline jobs/wegift_senior_python_software_engineer.html]: This line is needed!

**Comment by stevelittlefish** [comment]
I've fixed it.  I had a confusing merge conflict and thought the ---s were part of the markers git put in the file.

**Comment by salimfadhley** [comment]
All merged - thanks for using our site. Good luck filling the role.

**Pull request merged**